### ISSUE TITLE
Cannot use speaker sound under ipad (15.1)

### ISSUE THREAD

**Issue by zsinba** [jitsi/lib-jitsi-meet#1783]
*This Issue tracker is only for reporting bugs and tracking code related issues.*

Before posting, please make sure you check community.jitsi.org to see if the same or similar bugs have already been discussed. General questions, installation help, and feature requests can also be posted to community.jitsi.org.

## Description
---
Under the iPad, only a small sound can be heard, which comes from the earpiece;
The sound cannot be transmitted to the iPad's loudspeaker so that the sound is very small and can only be heard very close.

Our guess is that IOS bans Webrtc's external playback function to increase privacy. This is very suitable for voice calls, but not suitable for video calls.

## Current behavior
---
Use the web version or the libjs version to have this problem
## Expected Behavior
---
I can choose the sound output hardware

## Possible Solution
---
we can select the output device.
## Steps to reproduce
---
This is a must-have problem.

# Environment details
---
IOS 15.X

**Comment by saghul** [comment]
Are you not able to select the output device?

**Comment by zsinba** [comment]
may be about this : https://bugs.webkit.org/show_bug.cgi?id=230902

**Comment by zsinba** [comment]
> Are you not able to select the output device?

I don't think this has anything to do with it.

**Comment by saghul** [comment]
Yes, that could be it... or you are not using the output device and are instead using the earpiece. What device ID did you select on gUM?

**Comment by zsinba** [comment]
Updated the IOS version and made some improvements, but it didn't fix the problem.  I think it's IOS.

**Comment by zsinba** [comment]
Is there any progress on this?

**Comment by saghul** [comment]
You ever replied to the question about what device ID you selected in getUserMedia.
